components/playlist/browser/resources/components/playerControls.tsx

```diff
@@ -69,53 +68,45 @@ export default function PlayerControls ({ videoElementRef, className }: Props) {
     }
 
     return () => {}
-  }, [videoElementRef.current])
+  }, [videoElement])
 
   return (
     <Container className={className}>
       <div>
         <Control iconName='start-outline' onClick={() => {}}></Control>
         <Control
           iconName='rewind-15'
-          onClick={() => {
-            if (videoElementRef.current) {
-              videoElementRef.current.currentTime -= 15
-              if (videoElementRef.current.currentTime < 0)
-                videoElementRef.current.currentTime = 0
-            }
-          }}
+          onClick={() => videoElement && (videoElement.currentTime -= 15) }
         ></Control>
         {isPlaying ? (
           <Control
             iconName='pause-filled'
-            onClick={() => {
-              videoElementRef.current?.pause()
-            }}
+            onClick={() => videoElement?.pause()}
           ></Control>
         ) : (
           <Control
             iconName='play-filled'
-            onClick={() => {
-              videoElementRef.current?.play()
-            }}
+            onClick={() => videoElement?.play()}
           ></Control>
         )}
         <Control
           iconName='forward-15'
-          onClick={() => {
-            if (videoElementRef.current) {
-              videoElementRef.current.currentTime += 15
-            }
-          }}
+          onClick={() => videoElement && (videoElement.currentTime += 15)}
         ></Control>
         <Control iconName='end-outline' onClick={() => {}}></Control>
       </div>
       <div>
         <Control iconName='autoplay-off' onClick={() => {}}></Control>
         <Control iconName='shuffle-on' onClick={() => {}}></Control>
-        <Control iconName='sidepanel-open' onClick={() => {}}></Control>
+        {/* TODO(sko) this icon could be changed to sidepanel-open */}
+        <Control iconName='playlist-open-side' onClick={() => {}}></Control>
         <Control iconName='picture-in-picture' onClick={() => {}}></Control>
```

Comment:
I think there's a PiP function too, if you want another easy win (doesn't really matter though)
https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement/requestPictureInPicture